src/utils/make-build-info.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+from string import ascii_uppercase
+import logging
+import os
+
+import requests
+
+
+HEADER_FILE_TEMPLATE = """{license_header}
```

Comment:
Add warning about generated file and reference to file to modify (this one)

```suggestion
HEADER_FILE_TEMPLATE = """{license_header}

{generated_warning}
```
```
GENERATED_WARNING=""
/*
 * WARNING: This file is generated automatically.
 * Do not modify this file directly. 
 * Changes should be made in `src/utils/make-build-info.py`.
 */
"""